src/SearchResults.js

```diff
@@ -15,14 +15,13 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
-  // eslint-disable-next-line camelcase
-  const page_size = 10;
+  const limit = 10;
   const [
     searchOptionsForSearchResults,
     setSearchOptionsForSearchResults,
   ] = useState({
     ...SOUND_LIST_QUERY_PARAMS,
-    page_size,
+    page_size: limit,
```

Comment:
We renamed the variable denoting the number of results per page from `page_size` to `limit` [as amiljack suggested](https://github.com/amilajack/tide/pull/290#discussion_r546017123).

The name of the field used by the search options is still `page_size`, so we update its value using the newly renamed variable, `limit`, with `page_size: limit`.